File: C# Actividad POO/TestApp/ParqueApp/Parque.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ParqueApp {
    public class Parque {
        public string Nombre { get; set; }
        public string Ubicacion { get; set; }
        public double Superficie { get; set; }
        public List<Atraccion> Atracciones { get; set; }
        public List<Guardaparque> Guardaparques { get; set; }

        public Parque(string nombre, string ubicacion, double superficie) {
            Nombre = nombre;
            Ubicacion = ubicacion;
            Superficie = superficie;
            Atracciones = new List<Atraccion>();
            Guardaparques = new List<Guardaparque>();
        }

        public Parque()
        {
            Nombre = "Parque desconocido";
            Ubicacion = "Ubicacion desconocida";
            Superficie = 0;
            Atracciones = new List<Atraccion>(); 
            Guardaparques = new List<Guardaparque>(); 
        }

        public override string ToString() {
            return $"Parque(Nombre = {Nombre}, Ubicacion = {Ubicacion}, Superficie = {Superficie})";
        }
    }
}

File: ParquePY/parque.py
from atraccion import Atraccion
from guardaparque import Guardaparque

class Parque:
    def __init__(self, nombre="Parque Desconocido", ubicacion="Ubicacion Desconocida", superficie=0.0):
        self.nombre = nombre
        self.ubicacion = ubicacion
        self.superficie = superficie
        self.atracciones = []
        self.guardaparques = []

    def agregar_atraccion(self, atraccion):
        self.atracciones.append(atraccion)

    def agregar_guardaparque(self, guardaparque):
        self.guardaparques.append(guardaparque)

    def __str__(self):
        return (f"Parque(Nombre={self.nombre}, Ubicacion={self.ubicacion}, Superficie={self.superficie}, "
                f"Atracciones={len(self.atracciones)}, Guardaparques={len(self.guardaparques)})")
